### ISSUE TITLE
When will react-fiber be released officially?

### ISSUE THREAD

**Issue by shweta-code** [acdlite/react-fiber-architecture#18]
We want to start a new project . We did POC for about a month on choice of frameworks. We chose React for building our application. So just wanted to know if you got an idea on when will facebook officially release React-fiber because we dont want to be doomed.

**Comment by sateffen** [comment]
> React 16 release is expected this summer. But we remain committed to stability, just like in previous releases

[See here](https://twitter.com/reactjs/status/854989063027392512)

You can use React 15.5, and the update to 16 is just a drop in:

>  If your app produces zero warnings in 15.5, it should continue to work in 16 without any changes.

[See here](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html)

**Comment by wxg3a702** [comment]
React -fiber has not been released yet. How can you use it to create a Demo?

**Comment by danielkcz** [comment]
@wxg3a702 That is not correct. React Fiber is part of React 16 which is out for a couple of months already ;) 

https://reactjs.org/blog/2017/09/26/react-v16.0.html
